packages/core/admin/ee/admin/content-manager/components/Filter/CustomInputs/ReviewWorkflows/tests/ReviewWorkflowsFilter.test.js

```diff
@@ -8,7 +8,7 @@ import { setupServer } from 'msw/node';
 import { IntlProvider } from 'react-intl';
 import { QueryClientProvider, QueryClient } from 'react-query';
 
-import { ReviewWorkflowsFilter } from '../ReviewWorkflowsFilter';
+import { StageFilter } from '../StageFilter';
```

Comment:
Should this file be called `StageFilter.test.js` ?

Reply:
Good catch. Done & rebased.